pages/manual/planet.md

```diff
@@ -44,7 +44,7 @@ The **Nations** may store thousands of Resources in many formats and languages.
 **NOTE:** Each Planet system is unique; universal memberships do not exist. If you become a member of a specific Nation or Community, you do not automatically become a member of any other existing Nations and Community. To become a member of another Nation or Community you must register with a new account.
 
 ## Become Member
-To be able to use **PLANET** you need to be part of the system. First thing you 
+To be able to use **PLANET** you need to be part of the system. First thing you
```

Comment:
As you are removing extra space its better to not break this line as it is single sentence

Reply:
I added corrected the spacing and pushed it to branch.